### PR TITLE
fix: Fix some Geant4<->Acts unit conversions

### PR DESCRIPTION
**Pull request by andiwand**
Adds missing mass and time conversion factors

blocked by
- https://github.com/acts-project/acts/pull/2260

### CODE REVIEW

File: Examples/Algorithms/Geant4/src/ParticleTrackingAction.cpp [hunk 1]
```diff
@@ -102,15 +102,15 @@ void ActsExamples::ParticleTrackingAction::PostUserTrackingAction(
 ActsExamples::SimParticle ActsExamples::ParticleTrackingAction::convert(
     const G4Track& aTrack, SimBarcode particleId) const {
   // Unit conversions G4->::ACTS
-  constexpr double convertTime = Acts::UnitConstants::s / CLHEP::s;
```
Comment (Corentin-Allaire) [inline Examples/Algorithms/Geant4/src/ParticleTrackingAction.cpp:105]: Changing this from s to ns shouldn't make any difference here right ?

Comment (andiwand) [reply Examples/Algorithms/Geant4/src/ParticleTrackingAction.cpp:105]: correct I just aligned that with the other files and I think `ns` is more natural in our context

Comment (Corentin-Allaire) [reply Examples/Algorithms/Geant4/src/ParticleTrackingAction.cpp:105]: I fully agree with the change, I just wanted to be sure ! The rest seem fine by me. So let me just approve !

**Review by Corentin-Allaire — APPROVED** [review]
Great catch, thanks for the PR

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results
### :red_circle: Some tests have failed!
Please investigate the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/5848058)!

|status|job|
|------|---|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825355) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsWorkflow.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825354) |
|:red_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateTracks.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825353) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsCoreSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825352) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsTrkSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825351) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825350) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateOrthogonalSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825349) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateClusters.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825348) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsPersistifyEDM.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825347) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsKfRefitting.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825346) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsExtrapolationAlgTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825345) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsITkTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825344) |
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825343) |
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825342) |
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825341) |
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825340) |
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/30825339) |

**Pull request merged**